docker-compose.yml

```diff
@@ -125,6 +125,9 @@ services:
         BZIMPORT_BZ_API_KEY: ${BZIMPORT_BZ_API_KEY:?Variable BZIMPORT_BZ_API_KEY must be set.}
         JIRA_AUTH_TOKEN: ${JIRA_AUTH_TOKEN:?Variable JIRA_AUTH_TOKEN must be set.}
         JIRAFFE_AUTO_SYNC: ${JIRAFFE_AUTO_SYNC}
+        ET_URL: ${ET_URL}
+        PRODUCT_DEF_URL: ${PRODUCT_DEF_URL}
+        DASHBOARD_URL: ${DASHBOARD_URL}
```

Comment:
Should be added to `docker-compose.test.yml`

Reply:
Already done by @osoukup in #14